components/composer/src/Composer.svelte

```diff
@@ -70,6 +71,7 @@
   export let access_token: string = "";
 
   export let value: Message | void;
+  export let message_body: Message | void;
```

Comment:
I think we could rename `message_body` to `message` or `message_object`. As `message_body` seems like it refers to Message.body. Wdyt @heathergh ?

Reply:
Also, we have a prop called `template` which technically is the message body.

Reply:
Agreed! I changed it to `message_with_body`